Applications/VpView/vpVidtkTrackIO.cxx

```diff
@@ -973,12 +973,13 @@ void vpVidtkTrackIO::ReadTrack(
       timeStamp.SetTime(trackState->time_.time());
       }
 
-    if (trackState->data_.get(vidtk::tracking_keys::img_objs, objs))
+    obj = trackState->get_image_object();
+    if (obj != nullptr)
```

Comment:
Again, please modernize:
```suggestion
    if (auto const obj = trackState->get_image_object())
```
...and remove the prior declaration.

(Incidentally, I approve of getting rid of out parameters! Those are awful...)